description.md

```diff
@@ -54,6 +54,43 @@ This chapter describes the various technologies used to develop, store, deploy a
 
 ### Third Party (Open) Tools Employed
 
+As part of content development, delivery and consumption, third party tools may be used to enhance the teacher or user experience.
+
+Common examples for tools that projects integrate into their workflows:
+
+* file converters;
+* file generators;
+* content compilers;
+* content deployment workflows.
+
+#### When should you use a third party tool?
+
+* you need to automate something
+* you need external functionality that has already been implemented
+* you need to scale out a class use case that can only be done with mature tools
+
+#### What to look for in an Open Source project
+
+When choosing a tool to be employed during any phase of the class life cycle, one has to mind the following factors:
+
+* community involvement in product development;
+* development history;
+* adoption from other parties or institutions;
+* success stories with regards to use cases that are compatible with yours;
+
+Open source projects depend on the community to move forward development and design goals based on a stated common use case.
+The more community support a project has, the more the odds that it will be a long lived, well rounded solution.
+
+Open source projects can be easily tracked using issue trackers, and versioning systems to determine the maturity of the code base.
+Sporadic work on a project or a small person development team mean that the tool will be slow to integrate new features or to do bug fixing work.
+
+Institutional support for a software solution makes it so there is financial interest in developing the software.
+If a company or institution's financial outcome depends on a project, there will be more development or funding interest from this company.
+For example, when Meta has chosen to use a certain file deployment solution, it is likely for more bugs to be reported and fixed.
+
+Success stories can be a way to follow a well established example for using a software solution.
+A project can follow in another project's footsteps to gain an insight and to adopt compatible workflows.
```

Comment:
These last 2 paragraphs are void of practical usefulness. I would keep it more concrete: what tools do we use? what for? how should we use them? You can give concrete examples of our development stack for some of our deliverables such as vmchecker-next.

Reply:
Noted. I avoided doing this so that we keep the methodology top down and not IT education specific.